deploy/e2e-olm/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+resources:
```

Comment:
do you think, `deploy/production` or `deploy/osd` would be a better name/target than `e2e-olm` since this represents the production env ?